source/ui/win32/Win32Window.ooc

```diff
@@ -42,7 +42,6 @@ Win32Window: class extends NativeWindow {
 		super()
 	}
 	draw: func (image: RasterRgba) {
-		image swapRedBlue()
```

Comment:
If BGR was given to X11 then I have to assume that the same goes for Win32 that wants BGR format.